base-notebook/start-notebook.sh

```diff
@@ -15,10 +19,8 @@ if [[ "${RESTARTABLE}" == "yes" ]]; then
 fi
 
 if [[ -n "${JUPYTER_ENABLE_LAB}" ]]; then
```

Comment:
This should be change from `-n` to `-v` and if this variable is set at all, then we should warn the user it won't make any difference.

Reply:
Done 👍